contracts/contracts/ValidatorProxy.sol

```diff
@@ -0,0 +1,31 @@
+pragma solidity ^0.5.8;
+
+
+contract ValidatorProxy {
+
+    mapping (address => bool) public isValidator;
+    address public systemAddress = 0xffffFFFfFFffffffffffffffFfFFFfffFFFfFFfE;
+    address[] public validators;
+
+    function updateValidators(address[] memory newValidators) public {
```

Comment:
Do we have to support this function in the proxy? Or is this just for testing purposes?

Reply:
This is the proxy to the ValidatorSet that allows to have access to the set of validators on the chain even though the active ValidatorSet contract might change. This is not related to the bridge.